project/Build.scala

```diff
@@ -0,0 +1 @@
+case class SparkInfo(py4jVersion: String, breezeVersion: String)
```

Comment:
You can define anything you like in Build.scala and access it in the build.sbt script.